docs/SRIOV.md

```diff
@@ -0,0 +1,260 @@
+
+The following NICs have been tested with the Platform9 PMO 3.11.3 release:
+
+* Mellanox ConnectX-4 Lx EN
+* Mellanox ConnectX-5 EN
+* Intel X520
+* Intel X540-T2
+* Broadcom NetXtreme II (BCM57810 / HP 533FLR-T)
+
+The following drivers are considered supported:
+
+* ixgbe
+* bnx2x
+
+> Mellanox cards require additional configuration that is outside the scope of this guide and Platform9 Express.
+
+## Limitations
+
+The following are a few of the limitations of SR-IOV:
+
+* Bonded NICs at the host-level are not recommended/not supported for use with SR-IOV. While active/passive bonding may work in this configuration, LACP/802.3ad is definitely not a supported configuration.
+* Virtual Functions are automatically assigned to Neutron ports and are not customizable. 
+* Instance-level NIC bonding using Virtual Functions is not supported.
+* Port security/security groups are not supported.
+* VLAN networks are required. Flat (untagged) and overlay networks are not supported.
+
+## System Prerequisites
+
+SR-IOV requires the following:
+
+* BIOS Support (configuration varies by vendor)
+* Kernel IOMMU Support
+* Kernel IOMMU Passthrough support
+* Compatible Network Interface Card (NIC)
+
+> When SR-IOV capable NICs are used in conjunction with Open vSwitch bridges, you have the option of using an existing provider label, such as **external**, or using a dedicated provider. When sharing a provider network between SR-IOV and non-SR-IOV ports, communication between the ports on the same network is permitted. Using a dedicated provider will require you to call out a second bridge mapping, such as `sriov:br-sriov`, to allow DHCP ports connected to a vSwitch to communicate with the SR-IOV ports.
+
+### Kernel IOMMU Support
+
+Using **dmesg**, you can verify if IOMMU is enabled with the following command:
+
+```
+# dmesg | grep IOMMU
+```
+
+If you do not see the message ```DMAR: IOMMU enabled```, then proceed with the following steps:
+
+First, enable IOMMU support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... intel_iommu=on" #Intel-based Systems
+
+GRUB_CMDLINE_LINUX="... amd_iommu=on" #AMD-based Systems
+```
+
+Next, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+### IOMMU Passthrough Support
+
+To enable IOMMU passthrough support in the kernel, please complete the following steps:
+
+First, enable passthrough support in the kernel by modifying the GRUB configuration at **/etc/default/grub**:
+
+```
+GRUB_CMDLINE_LINUX="... iommu=pt"
+```
+
+Then, update GRUB:
+
+```
+update-grub
+```
+
+> Once the kernel configuration has been modified, you must reboot for the changes to take effect.
+
+## Deploying PMO with SR-IOV support using Express
```

Comment:
Is it worth this section being earlier or being mentioned earlier in the document so that the customer doesn't perform the grub updates manually that Express can handle?

Reply:
I changed the sentence to reflect this, but maybe it's better to just remove the manual steps altogether? Let me know.